src/components/Molecules/NavbarLeft/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+import Input from '../../atoms/Input';
+import Hamburguer from '../../atoms/Hamburguer';
+
+const NavBarLeftSC = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const handleChange = (event) => {
+  console.log(event);
```

Comment:
Just being curious, this handler is temporary, right? Waiting for final implementation of the `open/close` functionality? If so, it's common to write a `//TODO` comment e.g. `// TODO implement open/close functionality using component state, blah, blah...`